parrot/src/pfs_service_local.cc

```diff
@@ -326,7 +303,7 @@ process to sleep and wait for actual input to become ready.
 
 	virtual int is_seekable()
 	{
-		return !is_a_pipe;
```

Comment:
That issue no longer exists. All pipes/sockets/fifos are accessed via native file descriptors.